common/scala/build.gradle

```diff
@@ -91,6 +91,9 @@ dependencies {
         exclude group: 'com.fasterxml.jackson.dataformat'
     }
     compile ('com.amazonaws:aws-java-sdk-cloudfront:1.11.517')
+    compile ('com.azure:azure-storage-blob:12.0.0') {
+        exclude group: "com.azure", module: "azure-core-test"
```

Comment:
Opened Azure/azure-sdk-for-java#6207 to fix this

Reply:
@asteed This issue is closed now - can you update this PR to remove the exclude, and verify the extra dependencies are no longer pulled in?